xtuml/Events/models/Events/Events/Events/scenarios/scenarios.xtuml

```diff
@@ -1335,7 +1335,7 @@ INSERT INTO S_SYNC
 #          an event to test that events may be generated from services
 #-------------------------------------------------------------------------
 
-   [] = Events3::Generate_From_Service[The_Test_Number, firstX]
+   [] = Events3::Generate_From_Service[The_Test_Number, firstX.objX_id]
```

Comment:
Update to pass the instance ID and not the instance handle itself as mentioned in the previous file.